From: Antono Vasiljev <self@antono.info>
Date: Tue, 26 Jun 2012 11:58:09 +0300
Subject: Export NIX_REMOTE=daemon in nix-profile

---
 scripts/nix-profile.sh.in |    1 +
 1 file changed, 1 insertion(+)

diff --git a/scripts/nix-profile.sh.in b/scripts/nix-profile.sh.in
index 0c1e1bb..4f5b90c 100644
--- a/scripts/nix-profile.sh.in
+++ b/scripts/nix-profile.sh.in
@@ -8,6 +8,7 @@ if test -n "$HOME"; then
     fi
 
     export PATH=$NIX_LINK/bin:$PATH
+    export NIX_REMOTE="daemon"
 fi
 
 # This is a quick hack to make fontconfig-based packages in Nixpkgs
-- 
